cmd/render.go

```diff
@@ -39,7 +41,10 @@ func runRenderCmd(cmd *cobra.Command, args []string) {
 	if renderOpts.releaseImage == "" {
 		klog.Fatalf("missing --release-image flag, it is required")
 	}
-	if err := payload.Render(renderOpts.outputDir, renderOpts.releaseImage); err != nil {
+	if renderOpts.clusterProfile == "" {
+		klog.Fatalf("empty --cluster-profile flag, it is required")
```

Comment:
It should default to `default`, not be required.  And I see you set `default` above when declaring the `StringVar`.  So the error message should probably be `"invalid empty --cluster-profile value"` or some such.

Reply:
done :+1: